meson_options.txt

```diff
@@ -19,5 +19,5 @@ option('tracy_timer_fallback', type : 'boolean', value : false, description : 'U
 option('tracy_delayed_init', type : 'boolean', value : false, description : 'Enable delayed initialization of the library (init on first call)')
 option('tracy_manual_lifetime', type : 'boolean', value : false, description : 'Enable the manual lifetime management of the profile')
 option('tracy_fibers', type : 'boolean', value : false, description : 'Enable fibers support')
-option('tracy_shared_libs', type : 'boolean', value : false, description : 'Builds Tracy as a shared object')
+option('tracy_shared_libs', type : 'boolean', value : false, description : 'Builds Tracy as a shared object', deprecated : 'use default_library=shared instead')
```

Comment:
This argument to `deprecated` actually says which option meson should set instead. So currently it errors out:

```
tracy| DEPRECATION: Option 'tracy_shared_libs' is replaced by 'use default_library=shared instead'

subprojects/tracy/meson.build:1:0: ERROR: Tried to set unknown builtin option tracy:use default_library=shared instead
```

Reply:
the updated version with `deprecation: 'default_library'` has an issue where specifying `-Dtracy_shared_libs=true` is now an error as `bool` cannot match into `Default library type`. is this acceptable or should i use the simple bool deprecation flag and put the deprecation notice in the description?

Reply:
Hm, I think simple bool with deprecation and notice in description would be better.